src/mrpro/operators/_FourierOp.py

```diff
@@ -85,38 +85,36 @@ def __init__(
         nufft_dims = []
         fft_dims = []
         ignore_dims = []
-        fft_s = []
+        fft_recon_shape = []
```

Comment:
this already starts to look a lot like having a domain and a range for the operator ;)